src/platform/brand-consolidation/components/CallVBACenter.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import isBrandConsolidationEnabled from '../feature-flag';
+
+export default function CallVBACenter({ children, startSentence }) {
+  if (!isBrandConsolidationEnabled()) {
```

Comment:
Do we still need to include `!isBrandConsolidationEnabled()` clauses?

Reply:
Technically, no, but I wrote this before we launched and I'd like to leave the wbc clean up to a separate issue. There are several pieces of that that affect the components in this PR.